src/db/data-source.ts

```diff
@@ -0,0 +1,37 @@
+import 'dotenv/config';
+import 'reflect-metadata';
+import { DataSource, DataSourceOptions } from 'typeorm';
+
+const { NODE_ENV, DB_HOST, DB_PORT, DB_USERNAME, DB_PASSWORD, DB_DATABASE, DB_SSL } = process.env;
+
+const dataSourceOpts: DataSourceOptions = {
+    type: 'postgres',
+    host: DB_HOST,
+    port: parseInt(DB_PORT || '5432', 10),
+    username: DB_USERNAME,
+    password: DB_PASSWORD,
+    database: DB_DATABASE,
+    ssl: DB_SSL !== 'false' ?? true,
+    synchronize: false,
+    logging: false,
+    entities: [`${__dirname}/../entities/*.{ts,js}`],
+    migrations: [`${__dirname}/../migrations/*.{ts,js}`]
+};
+
+const { TEST_DB_HOST, TEST_DB_PORT, TEST_DB_USERNAME, TEST_DB_PASSWORD, TEST_DB_DATABASE } = process.env;
```

Comment:
I can't help feeling this violates the rule on not poluting our code with test code when we can load in a different configuration when doing the tests.

Reply:
Yeah, there's probably a better way of doing this, will revisit in another pass. Just needed something that would get postgres working in local/ci tests for now.